src/staking-v3/hooks/useAprV3.ts

```diff
@@ -0,0 +1,103 @@
+import { $api } from 'boot/api';
+import { ethers } from 'ethers';
+import { ref, watch } from 'vue';
+import { EraLengths } from '../logic';
+import { useDappStaking } from './useDappStaking';
+
+interface InflationParam {
```

Comment:
Move to https://github.com/AstarNetwork/astar-apps/tree/feat/dapp-staking-v3/src/staking-v3/logic/models, since all models are defined there

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1113/commits/eb7b59c9d4d18718e7be2e209bedcf449439ff26) commit.